packages/quantic/force-app/main/default/lwc/quanticTabBar/__tests__/quanticTabBar.test.js

```diff
@@ -0,0 +1,337 @@
+      expect(visibleTabs.length).toBe(expectedNumberOfTabsToBeVisible);
+
+      const expectedNumberOfTabsToBeHidden = 2;
+      const hiddenTabs = exampleTabSlots.filter(
+        (tab) => tab.style.visibility === 'hidden'
+      );
+      expect(hiddenTabs.length).toBe(expectedNumberOfTabsToBeHidden);
+    });
+
+    it('should display the correct tabs in the tabs dropdown list', async () => {
+      const element = createTestComponent(defaultOptions, exampleTabSlots);
+      await flushPromises();
+
+      const tabItemsInDropdown = element.shadowRoot.querySelectorAll(
+        selectors.tabItemsInDropdown
+      );
+
+      const expectedNumberOfTabsToBeVisible = 1;
+      const expectedNumberOfTabsToBeHidden = 2;
+
+      expect(tabItemsInDropdown.length).toBe(expectedNumberOfTabsToBeHidden);
+      const tabsInDropdownLabels = Array.from(tabItemsInDropdown).map(
+        (tab) => tab.textContent
+      );
+      const expectedTabsInDropdownLabels = exampleTabSlots
+        .slice(expectedNumberOfTabsToBeVisible)
+        .map((tab) => tab.label);
+      expect(tabsInDropdownLabels).toEqual(expectedTabsInDropdownLabels);
+    });
+
+    it('should open and close the tabs dropdown list after clicking the more tabs button', async () => {
+      const element = createTestComponent(defaultOptions, exampleTabSlots);
+      await flushPromises();
+
+      const moreTabsButton = element.shadowRoot.querySelector(
+        selectors.moreTabsButton
+      );
+      const tabsDropdown = element.shadowRoot.querySelector(
+        selectors.tabsDropdown
+      );
+      const expectedOpentabsDropdownCSSClass = 'slds-is-open';
+
+      expect(moreTabsButton).not.toBeNull();
+      expect(tabsDropdown).not.toBeNull();
+      expect(tabsDropdown.classList).not.toContain(
+        expectedOpentabsDropdownCSSClass
+      );
+
+      await moreTabsButton.click();
+      expect(tabsDropdown.classList).toContain(
+        expectedOpentabsDropdownCSSClass
+      );
+
+      await moreTabsButton.click();
+      expect(tabsDropdown.classList).not.toContain(
+        expectedOpentabsDropdownCSSClass
+      );
+    });
+
+    it('should call the select method of a tab when selecting it from the tabs dropdown list', async () => {
+      const element = createTestComponent(defaultOptions, exampleTabSlots);
+      await flushPromises();
+
+      const expectedNumberOfTabsToBeVisible = 1;
+      const expectedNumberOfTabsToBeHidden = 2;
+
+      const tabItemsInDropdown = element.shadowRoot.querySelectorAll(
+        selectors.tabItemsInDropdown
+      );
+
+      const exampleIndex = 0;
+      await tabItemsInDropdown[exampleIndex].click();
+      expect(tabItemsInDropdown.length).toBe(expectedNumberOfTabsToBeHidden);
+      expect(functionMocks.select).toHaveBeenCalledTimes(1);
+      expect(functionMocks.select).toHaveBeenCalledWith(
+        expectedNumberOfTabsToBeVisible + exampleIndex
+      );
+    });
+
+    describe('reactivity to window resize', () => {
```

Comment:
You already have this describe on line 143, I think these tests should be moved in the same describe group.